setup.py

```diff
@@ -64,5 +64,5 @@
             'medusa=medusa.medusacli:cli',
         ]},
     scripts=['bin/medusa-wrapper'],
-    data_files=[('/etc/medusa', ['medusa-example.ini'])]
+    data_files=[('etc', ['medusa-example.ini'])]
```

Comment:
This is actually being used during the installation phase to place the example in file into `/etc/medusa`. 
What's the behavior with this change?

Reply:
Hi @adejanovski ,

I think I got into python issue not medusa. I had an issue trying to install medusa wheel.
pip was installing medusa-example.ini into absolute /etc/medusa path even for virtual environment.
I tried to reproduce it today and medusa-example.ini was installed into site-packages as expected.